---
int32 result
float32 x
float32 y
float32 z
float32 r
float32[] jointAngle
